Library/Formula/gcc.rb

```diff
@@ -210,6 +216,17 @@ def install
       rmdir lib64
       prefix.install_symlink "lib" => "lib64"
     end
+    
+    if OS.linux?
+      p = @link.parent
+      @link.delete
+      p.delete
+      crts = Pathname.new "#{lib}/gcc/x86_64-unknown-linux-gnu/#{version}"
+      Formula['glibc'].lib.children.select {|p| p.basename.to_s =~ /^crt.\.o$/ }.collect {|p| p.relative_path_from crts}.each do |p|
+        crts.install_symlink p 
+      end
+    end
```

Comment:
xgcc automatically looks in that location for certain lib files. It is related to the -B and GCC_EXEC_PREFIX. I couldn't figure out how to pass command line flags to xgcc so that was how I made those files available to it.